backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -56,12 +56,14 @@ private static IQueryable<Entity.Views.Property> GenerateCommonQuery(this PimsCo
                 query = query.Where(p => p.RentableArea <= filter.RentableArea);
 
             if (filter.BareLandOnly == true)
-                query = (from p in query
-                         join pb in context.ParcelBuildings
-                            on p.Id equals pb.ParcelId into ppbGroup
-                         from pb in ppbGroup.DefaultIfEmpty()
-                         where pb == null && p.PropertyTypeId == Entity.PropertyTypes.Land
-                         select p);
+            {
+                var parcelIds = context.Parcels
```

Comment:
This will make a separate request for every land in the db were it has no associated building.  Not ideal if we can avoid it.  I'll provide a possible alternate later.